kombu/transport/virtual/base.py

```diff
@@ -177,6 +177,8 @@ def __init__(self, channel, prefetch_count=0):
         self.channel = channel
         self.prefetch_count = prefetch_count or 0
 
+        # Because we set an attribute on the OrderedDict, which
+        # is not possible with an ordinary dictionary
```

Comment:
Please rephrase the comment as it isn't using proper English.

Reply:
Is this better? I'd prefer not to trigger pipelines just for this.

```
        # Standard Python dictionaries do not support setting attributes
        # on the object, hence the use of OrderedDict
```

Reply:
@thedrow

Reply:
I pushed that change.